### DISCUSSION TITLE
Photo mode: can you set a single photo to 1/2sec shutter?

### DISCUSSION THREAD

**Discussion by sinetwo** [gopro/labs#797] [General]
I can't seem to find the information on how you set the shutter speed for photos exactly. I know you can do 2 seconds with night mode, but is there a way to set 1/2 sec with gopro labs, and iso of max 100, in standard photo mode?

Any help would be appreciated, so I can set the correct QR code to experiment with this.

Thanks in advance.

**Comment by dnewman-gpsw** [comment]
Not in standard photo, which always biases to a short shutter (likely for hand held shooting), but you can get 0.5s exposures from Night photo mode.  
![image](https://github.com/gopro/labs/assets/10856084/5a3d71a8-bca6-4d44-ac26-dd255539afb0)

This sets Night Photo with an Auto Shutter (key), but then forces the exposure to be 1/2 seconds with (oMEXPQ=2.)
